.github/workflows/nix.yaml

```diff
@@ -34,12 +34,13 @@ jobs:
         uses: >- # Custom commit, last pinned at 2024-03-03.
           DeterminateSystems/magic-nix-cache-action@742d8701e54851d2136f71b61d31382a4f4d0788
 
-      - name: Mount bazel cache
-        uses: >- # v4.0.1
-          actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319
-        with:
-          path: "~/.cache/bazel"
-          key: ${{ runner.os }}-bazel-nix
+      # TODO(aaronmondal): Somehow this cache breaks CI.
```

Comment:
This is more than a formatting error. I am fine with merging this one, but we need to open a ticket for investigating why this cache broke CI.